blueocean-dashboard/src/main/js/components/RunMessageCell.jsx

```diff
@@ -1,15 +1,18 @@
 import React, { Component, PropTypes } from 'react';
 import Lozenge from './Lozenge';
+import { Link } from 'react-router';
 
 export default class RunMessageCell extends Component {
     propTypes = {
         run: PropTypes.object,
         t: PropTypes.func,
+        linkTo: PropTypes.string,
     };
 
     render() {
         const run = this.props.run;
         const t = this.props.t;
+        const linkTo = this.props.linkTo || '';
```

Comment:
Seems like if we needed to link in here we could just generate it from the run data. We cant do that with TableCell so we have to pass it in. Seems redundant here and just something else to remember to do.

Reply:
Given the fact that I removed the link from the TableCell,I thought that passing it along where the table is constructed made most sense as it was already generated. I can definitely generate the link from the run but I will also have to pass along the pipeline  because buildRunDetailsUrl needs it to generate the url. I am still not sure that would make it better in the long run. What do you think?